docs/GettingStarted.md

```diff
@@ -207,7 +207,7 @@ Available settings are:
 * ``enabled``: define if the ``tracer`` is enabled or not. If set to
   ``false``, the code is still instrumented but no spans are sent to the local
   trace agent.
-* ``default_service``: set the service name used when tracing application
+* ``sidekiq_service``: set the service name used when tracing application
```

Comment:
because if we accept the Rails configuration, it already has a `default_service`: https://github.com/DataDog/dd-trace-rb/blob/a4faabfc66dc17c43b953d49670741f1f81d0621/lib/ddtrace/contrib/rails/framework.rb#L23